packages/charts/react-charts/library/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -7,16 +7,52 @@ import { bin as d3Bin, extent as d3Extent, sum as d3Sum, min as d3Min, max as d3
 import { scaleLinear as d3ScaleLinear } from 'd3-scale';
 import { format as d3Format, precisionFixed as d3PrecisionFixed } from 'd3-format';
 import { DonutChartProps } from '../DonutChart/index';
-import { ChartDataPoint, ChartProps, LineChartPoints, VerticalBarChartDataPoint } from '../../types/DataPoint';
+import {
+  ChartDataPoint,
+  ChartProps,
+  HorizontalBarChartWithAxisDataPoint,
+  LineChartPoints,
+  VerticalStackedChartProps,
+  HeatMapChartData,
+  HeatMapChartDataPoint,
+  GroupedVerticalBarChartData,
+  VerticalBarChartDataPoint,
+  SankeyChartData,
+  LineChartLineOptions,
+} from '../../types/DataPoint';
+import { SankeyChartProps } from '../SankeyChart/index';
+import { VerticalStackedBarChartProps } from '../VerticalStackedBarChart/index';
+import { HorizontalBarChartWithAxisProps } from '../HorizontalBarChartWithAxis/index';
 import { LineChartProps } from '../LineChart/index';
-import { getNextColor } from '../../utilities/colors';
+import { AreaChartProps } from '../AreaChart/index';
+import { HeatMapChartProps } from '../HeatMapChart/index';
+import { DataVizPalette, getColorFromToken, getNextColor } from '../../utilities/colors';
+import { GaugeChartProps, GaugeChartSegment } from '../GaugeChart/index';
+import { GroupedVerticalBarChartProps } from '../GroupedVerticalBarChart/index';
 import { VerticalBarChartProps } from '../VerticalBarChart/index';
 import { findNumericMinMaxOfY } from '../../utilities/utilities';
-import type { Datum, Layout, PlotlySchema, PieData, PlotData, TypedArray } from '@fluentui/chart-utilities';
-import { isArrayOfType, isDate, isDateArray, isNumberArray } from '@fluentui/chart-utilities';
+import type {
+  Datum,
+  Layout,
+  PlotlySchema,
+  PieData,
+  PlotData,
+  SankeyData,
+  ScatterLine,
+  TypedArray,
+} from '@fluentui/chart-utilities';
+import {
+  isArrayOfType,
+  isArrayOrTypedArray,
+  isDate,
+  isDateArray,
+  isNumberArray,
+  isLineData,
+} from '@fluentui/chart-utilities';
 import { timeParse } from 'd3-time-format';
+import { curveCardinal as d3CurveCardinal } from 'd3-shape';
 
-interface SecondaryYAxisValues {
+interface ISecondaryYAxisValues {
```

Comment:
we are not using interface starting with I in v9

Reply:
fixed